docs_src/microservices/support/scheduler/Ch-Scheduler.md

```diff
@@ -0,0 +1,111 @@
+|host|The host targeted by the action when it activates|
+|port|The port on the targeted host|
+|httpMethod |Indicates which Http verb should be used for the REST endpoint.(Only using when type is REST|
+|path|The HTTP path at the targeted host for fulfillment of the action.(Only using when type is REST)|
+|Publisher|The client to publish the message. The client ID must be unique across all clients connecting to the same broker.(Only using when type is MQTT)|
+|topic|The topic to publish to.|
+|qos|Determines the quality of service level used to guarantee message delivery.(Only using when type is MQTT)|
+|keepAlive|Maximum time in seconds to allow the connection to the MQTT broker to remain idle.(Only using when type is MQTT)|
+|retained|Determines whether or not the MQTT broker retains the message last published.(Only using when type is MQTT)|
+|autoReconnect|Reconnect to the broker when the connection failure.(Only using when type is MQTT)|
+|connectTimeout|The time interval in seconds to allow a connect to complete.(Only using when type is MQTT)|
+
+## High Level Interaction Diagrams
+
+**Scheduler interval actions to expunge old and exported (pushed) records from Core Data**
+
+![image](EdgeX_CoreDataCleanUp.png)
+
+## Configuration Properties
+
+Please refer to the general [Configuration documentation](../../configuration/Ch-Configuration.md#configuration-properties) for configuration properties common to all services.
+
+### ScheduleIntervalTime
+|Property|Default Value|Description|
+|---|---|---|
+|ScheduleIntervalTime|500|the time, in milliseconds, to trigger any applicable interval actions|
+### Intervals/Intervals.Midnight
+Default intervals for use with default interval actions
+|Property|Default Value|Description|
+|---|---|---|
+|Name|midnight|Name of the every day at midnight interval|
+|Start|20180101T000000|Indicates the start time for the midnight interval which is a midnight, Jan 1, 2018 which effectively sets the start time as of right now since this is in the past|
+|Interval|24h|defines a frequency of every 24 hours|
+### IntervalActions.IntervalActions.ScrubAged
+Configuration of the core data clean old events operation which is to kick off every midnight
+|Property|Default Value|Description|
+|---|---|---|
+|Name|scrub-aged-events|name of the interval action|
+|Host|localhost|run the request on core data assumed to be on the localhost|
+|Port|59880|run the request against the default core data port|
+|Protocol|http|Make a RESTful request to core data|
+|Method|DELETE|Make a RESTful delete operation request to core data|
+|Target|core-data|target core data|
+|Path|/api/v2/event/age/604800000000000|request core data's remove old events API with parameter of 7 days |
+|Interval|midnight|run the operation every midnight as specified by the configuration defined interval|
+
+
+## API Reference
+[Support Scheduler API Reference](../../../api/supporting/Ch-APISupportingServicesScheduler.md)
```

Comment:
Good place for an `!!! edgey` note about the API changes for 2.0

Reply:
PR #448 is noted for the change. Should adding the same note here?
![image](https://user-images.githubusercontent.com/41656797/124208277-cb83a600-db19-11eb-96cf-9f525dc1a8cc.png)